### ISSUE TITLE
adapt-contrib-slider Unable to interact with slider on mobile screen readers

### ISSUE THREAD

**Issue by simondate** [adaptlearning/adapt-contrib-slider#158]
After doing some accessibility testing it was discovered that this question can't be completed by mobile screen readers. Since there is no way to select with just swiping.

I'm not sure if this requires a higher level of accessibility that Adapt doesn't adhere to? (I'm just reporting what was raised by accessibility testers.)

### Your environment
* version (AT/Framework) 5.15.2
*  Slider version 3.1.5
* which browser and its version  Chrome 97 - TalkBack
* device(s) + operating system(s) Google Pixel 3a - Android 10

https://user-images.githubusercontent.com/1676635/163163661-f76bab42-f35c-47df-9726-6142251e964b.mov
